src/box2d.rs

```diff
@@ -220,7 +220,6 @@ where
 {
     /// Inflates the box by the specified sizes on each side respectively.
     #[inline]
-    #[cfg_attr(feature = "unstable", must_use)]
```

Comment:
Servo benefits from these annotations don't they? Plus must_use is stable now, so maybe we should just remove the cfg_attr.

Reply:
Ah nice! I hadn't realized it was stable in function annotations now. I don't know whether servo uses the unstable feature.
I'll remove the cfg_attr.